governance.md

```diff
@@ -33,7 +33,7 @@ Qualification guidelines
 * Has submitted at least 5 PRs of significant scope that are correspondingly merged
 * Sponsored by 2 Approvers
 
-[Current Reviewers](https://github.com/spinnaker/spinnaker/blob/master/reviewers.md)
+[Current Reviewers](https://github.com/spinnaker/community/blob/master/membership.yml#L23)
```

Comment:
@dorbin What would be an appropriate link here? Linking to particular lines in a file could be brittle; is the membership "rendered" anywhere with a potentially more stable link?

Reply:
How about just linking to the file? The sections within it are pretty clear, and it's not that huge.

Reply:
```suggestion
[Current Reviewers](https://github.com/spinnaker/community/blob/master/membership.yml#L23) (see the `oss-reviewers` section
```